array/DNAm/preprocessing/loadDataGDS.r

```diff
@@ -34,7 +34,7 @@ library(IlluminaHumanMethylationEPICmanifest)
 library(IlluminaHumanMethylation450kanno.ilmn12.hg19)
 library(IlluminaHumanMethylation450kmanifest)
 library(devtools)
-devtools::load_all(path = "../functionsR")
+devtools::load_all(path = "~/BrainFANS/functionsR")
```

Comment:
Well done!

Reply:
Does this work though? What if BrainFANS is not located in the home directory of the user?